docs/developing-plugins/building-provider-integrations.md

```diff
@@ -41,6 +42,11 @@ The purpose of the `deploy:createProviderStacks` lifecycle is to take the basic
 the previous lifecycle and deploy the rough skeleton on the cloud providers infrastructure (without any functions
 or events) for the first time.
 
+#### `deploy:createDeploymentPackage`
+
+The whole service get's zipped up into one .zip file. Serverless will automatically apply exclude rules based on the
+used programming languages (you can always include previously excluded files and folders if you want to).
```

Comment:
But we don't automatically include/exclude based on a specific language. Imho this should be implemented in templates, but not in the packaging mechanism. See this comment for details: https://github.com/serverless/serverless/issues/1517#issuecomment-231380348